homeassistant/components/alarm_control_panel/strings.json

```diff
@@ -62,5 +62,11 @@
         }
       }
     }
+  },
+  "issues": {
+    "platform_integration_no_support": {
+      "title": "Platform support not supported",
+      "description": "The {platform} platform for the {domain} integration does not support platform setup.\n\nPlease remove it from your configuration and restart Home Assistant to fix this issue."
```

Comment:
Can we put the common strings in the common strings file?

https://github.com/home-assistant/core/blob/dev/homeassistant/strings.json